contracts/courtroom_test.go

```diff
@@ -313,13 +339,13 @@ func TestDoubleRegistration(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Register: expected no error, got %v", err)
 	}
-	backend.Commit()
+	commit(backend)
 	opts = bind.NewKeyedTransactor(serviceKey)
 	_, err = swearGame.Register(opts, clientAddr)
 	if err != nil {
 		t.Fatalf("Register: expected no error, got %v", err)
 	}
-	backend.Commit()
+	commit(backend)
 	registerPlayerCounter, err := swearGame.RegisteredPlayersCounter(&bind.CallOpts{})
 	if err != nil {
```

Comment:
The block counter is needed for the service promise.
"I will serve you within the next X blocks"
It is part of the service promise and I it is tested  at TestPromiseOk (TestOpenNoneValidClaimDueWaitToLong)